name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RaiffeisenBankgruppePDFExtractor.java

```diff
@@ -699,83 +771,124 @@ private void addAccountStatementTransactions()
 
     private <T extends Transaction<?>> void addTaxesSectionsTransaction(T transaction, DocumentType type)
     {
+        // If we have a gross reinvestment,
+        // we set a flag and don't book tax below.
+        transaction
+                .section("n").optional()
+                .match("^Zu (?<n>Lasten) .* \\-[\\.,\\d]+ [\\w]{3}.*$")
+                .match("^Belastung KESt bei aussch.ttungsgleichen Erträgen .*$")
+                .assign((t, v) -> type.getCurrentContext().put("noTax", "X"));
```

Comment:
BTW, the ```DocumentContext``` also supports boolean directly.

You could say ```type.getCurrentContext().putBoolean("noTax", true)``` and use ```getBoolean("noTax")``` which resolves to true only if there is a boolean true in the map.